<launch>
 <group ns="lidar1">
  <node name="lidar01" pkg="bluesea2" type="bluesea2_node" output="screen" >
    <param name="type" value="uart" />
    <param name="platform" value="LDS-50C" />
    <param name="port" value="/dev/ttyUSB0" />
    <param name="frame_id" value="map" />
    <rosparam param="rate_list">[ 230400, 256000 ]</rosparam>
    <param name="baud_rate" value="230400" />
    <param name="max_dist" value="50.0"/>
    <param name="raw_bytes" value="2"/>
    <param name="output_scan" value="true" />
    <param name="output_cloud" value="false"/>
    <param name="output_360" value="true"/>   
    <param name="mirror" value="true"/>
    <param name="from_zero" value="true"/>
    <param name="with_resample" value="true"/>
    <param name="resample_res" value="0.6"/>
    <param name="angle_patch" value="false"/>
    <param name="unit_is_mm" value="false"/>
    <param name="with_confidence" value="true"/>
    <param name="with_checksum" value="false"/>  
  </node>
 </group>
</launch>
